extensionsGallery-insider.json

```diff
@@ -3780,7 +3780,7 @@
 								},
 								{
 									"key": "Microsoft.AzDataEngine",
-									"value": ">=1.27.0"
+									"value": ">=1.28.0"
```

Comment:
The package.json still has 1.27.0 - you should update that as well if this requires core changes that are still in insiders